docs/releases/2.8.0.rst

```diff
@@ -29,82 +31,162 @@ Major Changes
 - Pootle FS
 
 
-Below we provide much more detail.  These are by no means exhaustive, view the
-`git log
-<https://github.com/translate/pootle/compare/2.7.6...master>`_ for complete
-information.
+Below we provide much more detail. These are by no means exhaustive, view the
+`git log <https://github.com/translate/pootle/compare/2.7.6...master>`_ for
+complete information.
 
 
 Details of changes
 ==================
 
-- MySQL now uses the mysqlclient database driver instead of MySQLdb, this is
-  Django's preferred database driver.
+- MySQL now uses the ``mysqlclient`` database driver instead of ``MySQLdb``,
+  this is Django's preferred database driver.
 - Undertook a security audit
-- Improved editor performance
 - JavaScript fixes addressing performance and memory leaks in the editor
-- Extensive review and fixes to RTL layout in the editor and browse views
+- Improved editor performance
+- Extensive review and fixes to RTL layout
 - Added pluggable search backend
 - Pootle FS:
 
+  - Added admin UI to set up projects configuration and language mapping
   - CLI
   - LanguageMapper
   - FileMapper
   - Store de/serialization
 
-- Improved UI/stats reporting
-- Support rejecting/accepting suggestions with comments
+- Refactored code for syncing stores
+- Changed the default ``robots.txt``
+
+  - It is now a static file
+  - Now the site is ``allow`` by default
+
+- Editor:
+
+  - Suggestions:
+
+    - Support rejecting/accepting suggestions with comments
+
+      - Suggesters are emailed based on new
+        :setting:`POOTLE_EMAIL_FEEDBACK_ENABLED` setting
+
+    - Accept suggestion without "can review" permissions if submitted
+      translation is 100% match.
+
+  - Special characters are depicted by symbols from a custom-built font:
+
+    - Properly display whitespace as special character
+    - Technical details of escape sequences are omitted in the output displayed
+      to end users
+
+  - Errors fixed:
+
+    - Fixed bugs in muting/unmuting checks
+    - Fixed filtering translations by month
+    - Fixed ``TypeError`` error when filter in editor gets no units
+
+  - Translation memory:
+
+    - Display original and translations side-by-side for TM results
+    - amaGama Translation Memory is now queried using CORS
+
+  - Perform all highlighting in the client
+  - Clearly present plurals to translator
+  - Allow selecting similar translations' text in editor
+  - Force word wrapping on long strings with no spaces
+  - Improvements on timeline
+  - Cross-language translation is now restricted to admins since translators
+    don't require them and it heavily impacts performance.
```

Comment:
I'd reword, it sounds like a very opinionated assumption,  "Cross-language translation is now restricted to admins as they heavily impacts performance and translators are unlikely to require them"